.danger.php

```diff
@@ -43,78 +41,21 @@
             $context->warning('The Pull Request doesn\'t contain any changelog file');
         }
     })
-    ->useRule(new Condition(
-        function (Context $context) {
-            return $context->platform instanceof Gitlab;
-        },
-        [
-            function (Context $context): void {
-                $labels = array_map('strtolower', $context->platform->pullRequest->labels);
-
-                if ($context->platform->raw['squash'] === true && in_array('github', $labels, true)) {
-                    $context->failure('GitHub PRs are not allowed to be squashed');
-                }
-            },
-        ]
-    ))
-    /**
-     * MRs that target a release branch that is not trunk should have a thread with link to a trunk MR
```

Comment:
@shopware/product-operations not sure, if we still need references to other PRs?

Reply:
not needed